README.md

```diff
@@ -61,3 +61,17 @@ jobs:
 See the individual Actions linked above for per-language instructions.
 
 Note: GitHub Actions will not pass on secrets set in the repository to forks being used in pull requests, and so the Snyk actions that require the token will fail to run.
+
+## Getting your Snyk token
+
+The Actions example above refer to a Snyk API token:
+
+```
+      env:
+        SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+```
+
+Every Snyk account has this token, and you can find it in one of two ways:
+1. If you're using the [Snyk CLI](https://support.snyk.io/hc/en-us/articles/360003812458-Getting-started-with-the-CLI) you can retrieve it by running `snyk config get api`
```

Comment:
Missing a full spot. The second list item has one. Best to be consistent. Either add a full stop on this line or remove from the one below